docs/guide/yaml/yaml-reference.md

```diff
@@ -37,7 +37,10 @@ the entity being defined, with these being the most common:
 
 * `brooklyn.policies`: a list of policies, each as a map described with their `type` and their `brooklyn.config` as keys
 
-* `brooklyn.enrichers`: a list of enrichers, each as a map described with their `type` and their `brooklyn.config` as keys
+* `brooklyn.enrichers`: a list of enrichers, each as a map described with their `type` and their `brooklyn.config` as keys;
+  see the keys declared on individual enrichers; 
+  also see [this enricher example](example_yaml/test-app-with-enrichers-slightly-simpler.yaml) for a detailed and commented illustration
```

Comment:
Would definitely like a different name - the "slightly simpler" should just be "normal" in the eyes of the user, so is redundant and confusing in the name.